examples/classic/distributed/ClockSync/clockSync3.qnt

```diff
@@ -139,3 +139,11 @@ module ClockSync3 {
         }
     }
 }
+
+// The default instance of the clockSync spec
```

Comment:
This is the only approach I’m aware of to make constant parameterized tests runnable. Please let me know if I’m missing a better approach.

Reply:
I think this is currently the only way